.github/workflows/inspektor-gadget.yml

```diff
@@ -2048,7 +2048,7 @@ jobs:
   test-integration-gke:
     name: Integration tests on GKE
     # level: 4
-    if: needs.check-secrets.outputs.gke == 'true'
+    if: needs.check-secrets.outputs.gke == 'true' && false
```

Comment:
How about we remove the secrets from the repository?

We have them documented, in case we want to enable GCP again: https://inspektor-gadget.io/docs/latest/devel/ci#run-integration-tests-on-gke

Reply:
Indeed, it may be better to proceed as you suggest, as in any case we would need to create these secrets again.
The main idea with this PR was to avoid running it for nothing while not removing the code, we could still revert in case we want to activate it again though.